lib/appsignal_phoenix/event_handler.ex

```diff
@@ -8,8 +8,8 @@ defmodule Appsignal.Phoenix.EventHandler do
 
   def attach do
     handlers = %{
-      [:phoenix, :endpoint, :start] => &__MODULE__.phoenix_endpoint_start/4,
-      [:phoenix, :endpoint, :stop] => &__MODULE__.phoenix_endpoint_stop/4,
+      [:phoenix, :router_dispatch, :start] => &__MODULE__.phoenix_router_dispatch_start/4,
+      [:phoenix, :router_dispatch, :stop] => &__MODULE__.phoenix_router_dispatch_stop/4,
```

Comment:
Would it be interesting to instrument both events (start and stop) events? That way people could see how long their app takes from the router to the endpoint, and measure whatever else goes on before then.